dockstore-client/src/main/java/io/dockstore/client/cli/nested/WESLauncher.java

```diff
@@ -114,7 +115,9 @@ public void runWESCommand(String jsonInputFilePath, File localPrimaryDescriptorF
         try {
             RunId response = clientWorkflowExecutionServiceApi.runWorkflow(workflowParams, languageType, workflowTypeVersion, TAGS,
                     "{}", workflowURL, workflowAttachment);
-            System.out.println("Launched WES run with id: " + response.toString());
+            String runID = response.getRunId();
+            System.out.println("Launched WES run with id: " + runID);
```

Comment:
Here you use `System.out.println`, starting on line 137 you use `out` -- should be consistent.

Reply:
now using out only